core/Error.h

```diff
@@ -92,17 +92,16 @@ class Error {
  * Used to batch errors in an RAII fashion:
  *
  * {
- *   ErrorRegion errs(gs);
+ *   ErrorRegion errs(fref);
  *   runNamer();
  * }
  */
 class ErrorRegion {
 public:
-    ErrorRegion(const GlobalState &gs, FileRef f) : gs(gs), f(f){};
+    ErrorRegion(FileRef f) : f(f){};
```

Comment:
I didn't realize this before, but it looks like ErrorRegion's only purpose before was to batch report errors. We can completely remove it.

Reply:
Yup, that was the reason :-)